sources/packages/web/src/main.ts

```diff
@@ -52,6 +52,7 @@ import {
   faFolderOpen,
   faArrowLeft,
   faConciergeBell,
+  faGraduationCap,
 } from "@fortawesome/free-solid-svg-icons";
```

Comment:
Not a part of this story, we need to add a point in any of the technical debt to import free regular icons because some of the icons I saw from figma are free regular ones.

Reply:
I actually tried it in this story for faGraduationCap, but regular faGraduationCap was not in free regular ones, so I reverted the changes

Reply:
Yes, but I have seen few icons, couldn't recollect which ones but they were from free regular.